apps/meeting/src/components/MeetingControls/VideoInputTransformControl.tsx

```diff
@@ -0,0 +1,238 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Remove `Apache-2.0` from all files in this PR. This sample repo uses MIT-0.
https://github.com/aws-samples/amazon-chime-sdk/blob/main/LICENSE

Reply:
done